content/hi/docs/concepts/overview/kubernetes-api.md

```diff
@@ -0,0 +1,241 @@
+        <td><code>application/json</code></td>
+        <td><em>default</em></td>
+     </tr>
+     <tr>
+        <td><code>*</code></td>
+        <td><em>serves </em><code>application/json</code></td>
+     </tr>
+  </tbody>
+</table>
+
+कुबेरनेट्स एक वैकल्पिक प्रोटोबॉफ़ आधारित क्रमांकन प्रारूप को लागू करता है
+मुख्य रूप से इंट्रा-क्लस्टर संचार के लिए अभिप्रेत है। अधिक जानकारी के लिए
+इस प्रारूप के बारे में, [Kubernetes Protobuf क्रमांकन](https://git.k8s.io/design-proposals-archive/api-machinery/protobuf.md) डिज़ाइन प्रस्ताव और देखें
+गो में स्थित प्रत्येक स्कीमा के लिए इंटरफेस डेफिनिशन लैंग्वेज (आईडीएल) फाइलें
+संकुल जो एपीआई वस्तुओं को परिभाषित करता है।
+
+### ओपनएपीआई V3
+
+{{< फीचर-स्टेट स्टेट = "बीटा" for_k8s_version = "v1.24">}}
+
+Kubernetes {{< param "संस्करण">}} अपने API को OpenAPI v3 के रूप में प्रकाशित करने के लिए बीटा समर्थन प्रदान करता है; यह है एक
+बीटा सुविधा जो डिफ़ॉल्ट रूप से सक्षम है।
+आप बंद करके बीटा सुविधा को अक्षम कर सकते हैं
+[फीचर गेट](/डॉक्स/संदर्भ/कमांड-लाइन-टूल्स-रेफरेंस/फीचर-गेट्स/) जिसका नाम `OpenAPIV3` है
+क्यूब-एपिसर्वर घटक के लिए।
+
+सभी की सूची देखने के लिए एक डिस्कवरी एंडपॉइंट `/openapi/v3` प्रदान किया गया है
+समूह/संस्करण उपलब्ध हैं। यह समापन बिंदु केवल JSON लौटाता है। ये समूह/संस्करण
+निम्नलिखित प्रारूप में प्रदान किए गए हैं:
+
+```yaml
+{
+    "paths": {
+        ...,
+        "api/v1": {
+            "serverRelativeURL": "/openapi/v3/api/v1?hash=CC0E9BFD992D8C59AEC98A1E2336F899E8318D3CF4C68944C3DEC640AF5AB52D864AC50DAA8D145B3494F75FA3CFF939FCBDDA431DAD3CA79738B297795818CF"
+        },
+        "apis/admissionregistration.k8s.io/v1": {
+            "serverRelativeURL": "/openapi/v3/apis/admissionregistration.k8s.io/v1?hash=E19CC93A116982CE5422FC42B590A8AFAD92CDE9AE4D59B5CAAD568F083AD07946E6CB5817531680BCE6E215C16973CD39003B0425F3477CFD854E89A9DB6597"
+        },
+        ....
+    }
+}
+```
+
+<!-- संपादकों के लिए: सिंटैक्स हाइलाइट त्रुटि को रोकने के लिए जानबूझकर यहां json के बजाय yaml का उपयोग करें। -->
+
+सापेक्ष URL अपरिवर्तनीय OpenAPI विवरणों की ओर इशारा कर रहे हैं
+क्लाइंट-साइड कैशिंग में सुधार करने के लिए। उचित HTTP कैशिंग हेडर
+उस उद्देश्य के लिए एपीआई सर्वर द्वारा भी निर्धारित किया जाता है (`समाप्ति` से 1 वर्ष में
+भविष्य, और `कैश-कंट्रोल` से `अपरिवर्तनीय`)। जब एक अप्रचलित URL है
+प्रयुक्त, एपीआई सर्वर नवीनतम यूआरएल पर रीडायरेक्ट लौटाता है।
+
+कुबेरनेट्स एपीआई सर्वर प्रति कुबेरनेट्स में एक ओपनएपीआई v3 विशिष्टता प्रकाशित करता है
+`/openapi/v3/apis/<group>/<version>?hash=<hash>` पर समूह संस्करण
+समापन बिंदु।
+
+स्वीकृत अनुरोध शीर्षलेखों के लिए नीचे दी गई तालिका देखें।
+
+<table>
+  <caption style="display:none">Valid request header values for OpenAPI v3 queries</caption>
+  <thead>
+     <tr>
+        <th>Header</th>
+        <th style="min-width: 50%;">Possible values</th>
+        <th>Notes</th>
+     </tr>
+  </thead>
+  <tbody>
+     <tr>
+        <td><code>Accept-Encoding</code></td>
+        <td><code>gzip</code></td>
+        <td><em>not supplying this header is also acceptable</em></td>
```

Comment:
You may want to translate the table contents as well.

Reply:
Ohk sure

Reply:
For all the table content?